spec/views/user_show_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+RSpec.describe 'User Page show', type: :feature do
+  before :each do
+    @user = User.create(
+      name: 'Tommy',
+      photo: 'https://images.unsplash.com/photo-1545167622-3a6ac756afa4?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=712&q=80',
+      bio: 'Teacher',
+      posts_counter: 3
+    )
+    @post1 = Post.create(title: 'Hello',
+                         text: 'My world',
+                         comments_counter: 0,
+                         likes_counter: 0,
+                         author_id: @user.id)
+    @post2 = Post.create(title: 'Hi',
+                         text: 'My earth',
+                         comments_counter: 0,
+                         likes_counter: 0,
+                         author_id: @user.id)
+    @post3 = Post.create(title: 'Hey',
+                         text: 'My universe',
+                         comments_counter: 0,
+                         likes_counter: 0,
+                         author_id: @user.id)
+    visit user_path(@user.id)
+  end
+
+  it 'shows the profile picture of the user' do
+    assert page.has_xpath?("//img[@src = 'https://images.unsplash.com/photo-1545167622-3a6ac756afa4?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=712&q=80'
+     and @alt='user photo']")
```

Comment:
- [ ] NIce adding the required changes by the previous reviewer however there are some tests that are failing please make sure to fix that in order to complete the requirements 
<img width="841" alt="Screenshot 2023-02-15 at 16 08 51" src="https://user-images.githubusercontent.com/61790587/219067015-91794e31-2371-4867-b04c-88a2ca92ff25.png">